examples/.gitignore

```diff
@@ -0,0 +1,3 @@
+cpp/**/CMakeLists.txt
```

Comment:
Should not need these ignored. CMake files should be generated into the binary for.